src/rules/obsolete_statement.ts

```diff
@@ -63,13 +63,29 @@ export class ObsoleteStatement extends ABAPRule {
         prev = sta.getStart();
       }
 
+      if ((sta.get() instanceof Statements.Describe)
```

Comment:
please also add as setting in ObsoleteStatementConf

Reply:
I did the check similar to "requested" - I didn't even see that "requested" has a config element, because it currently isn't being checked! Will add that, too.